spec/factories/user.rb

```diff
@@ -28,5 +28,10 @@
       authentication_token { SecureRandom.hex }
       authentication_token_created_at { Time.now - 1.day }
     end
+
+    trait :without_created_at do
```

Comment:
Please disambiguate here and call the trait `without_authentication_token_created_at`.